build_tools/build_requirements.txt

```diff
@@ -1,4 +1,5 @@
-numpy~=1.19.3
+numpy~=1.19.3; python_version < '3.10'
+numpy~=1.21.2; python_version >= '3.10'
```

Comment:
This is cool, didn't know of this syntax

Reply:
Yeah, I didn't either! In this case it is because numpy doesn't distribute for 3.10 on 1.19.X, and we need to pin the lowest supported version. It also works with things like platform and whatnot

https://pip.pypa.io/en/stable/cli/pip_install/#requirement-specifiers